.github/scripts/cherry_pick_check.ps1

```diff
@@ -0,0 +1,15 @@
+param([string]$commitMessage)
+
+#Looks for the cherry-pick command on the commit message
+$commitData = $commitMessage | Select-String -Pattern "Cherry-pick to:\s?'*'"
+
+#Extracts the branch name
+if($commitData){
+    $splitedData = $commitData.Line.split("'")
+    $branchName = $splitedData[1]
```

Comment:
So this means the Cherry-pick to `branchname` needs to be in the first line of the commit message right? Otherwise, other commit message may rune this condition, would you add comments?
e.g.
* commit message add resource string 'xxx'
* ...

Cherry-pick to `target branch`

Reply:
The "cherry-pick to: 'branch'" command is captured on line 4 and currently it can be in any part of the description. The lines you selected (8,9) are the ones that extract the branch name from the command. Maybe "commitData" isn't the best name for that variable as it can generate some confusion.

Reply:
Oh I see, so the `commitData` will be the string `Cherry-pick to 'target branch'`? Would you add the format as comment on line 4?

Reply:
That is correct, added the comment and changed that variable name for something more representative of its value